marketstack/adapter.js

```diff
@@ -0,0 +1,43 @@
+const { Requester, Validator } = require('@chainlink/external-adapter')
+const API_KEY = process.env.API_KEY
+const DEFAULT_INTERVAL = '1min'
+const DEFAULT_LIMIT = 1
+
+const customParams = {
+  base: ['base', 'from', 'asset'],
+  endpoint: false,
+  interval: false
+}
+
+const createRequest = (input, callback) => {
+  const validator = new Validator(callback, input, customParams)
+  const jobRunID = validator.validated.id
+  const endpoint = validator.validated.data.endpoint || 'intraday'
```

Comment:
Maybe introduce a `DEFAULT_ENDPOINT` to follow the convention you've made here?

Reply:
makes sense